terraform/Development/main.tf

```diff
@@ -17,11 +17,6 @@ locals {
    parameter_store = "arn:aws:ssm:${data.aws_region.current.name}:${data.aws_caller_identity.current.account_id}:parameter"
 }
 
-
-data "aws_iam_role" "ec2_container_service_role" {
-  name = "ecsServiceRole"
-}
-
 data "aws_iam_role" "ecs_task_execution_role" {
   name = "ecsTaskExecutionRole"
 }
```

Comment:
Same for this that Emma pointed out for the EC2 role.
The task execution role isn't needed for Lambda either - in Staging/Prod or Development.